docs/cmdline_migration.md

```diff
@@ -39,6 +39,7 @@ You can use the following command-line options in OpenJ9, just as you did in Hot
 | [`-Xcheck:jni`](xcheck.md)                                       | Runs additional checks for JNI functions during VM startup.                                                                                  |
 | [`-Xfuture`](xfuture.md)                                         | Turns on strict class-file format checks.                                                                                                    |
 | [`-Xint`](xint.md)                                               | Runs an application in interpreted-only mode.                                                                                                |
+| [`Xlog`](xlog.md)                                                | Some forms of `-Xlog` which enable GC logging are recognized. (Equivalent to [`-Xverbosegclog`](xverbosegclog.md)).
```

Comment:
Link name missing "-" ........ `-Xlog`

Reply:
-> Some forms of `-Xlog` *that* enable GC logging are recognized.

Reply:
Updated to fix both.